viewing_party/party.py

```diff
@@ -0,0 +1,117 @@
+def create_movie(movie_title, genre, rating):
+    new_movie = { }
+    if movie_title==None or genre==None or rating==None:
+        return None
+    new_movie['title'] = movie_title
+    new_movie['genre'] = genre
+    new_movie['rating'] = rating
+    return new_movie
```

Comment:
This works great but you could just return the title. genre, and rating without doing new_movie = {}

```suggestion
def create_movie(movie_title, genre, rating):
    if movie_title == None or genre == None or rating == None:
        return None
    else:
        return {
        "title":title, 
        "genre":genre, 
        "rating":rating
        }
```